sites/mosul/configs/openmrs/initializer_config/ampathforms/F08-ITFC_Admission_form.json

```diff
@@ -1291,8 +1291,8 @@
               }
             },
             {
-              "id": "secondary1",
-              "label": "Secondary 1",
+              "id": "diagnosisAtAdmissionSecondary1",
+              "label": "Diagnosis at admission - Secondary 1",
```

Comment:
The `Label if different` shows only the removed texts as the label.

<img width="883" alt="image" src="https://github.com/user-attachments/assets/dc5e5168-a5da-43ba-82b7-158973515147" />

Reply:
Yes, the skip logic was resolved a while back but there were conflicts with duplicated ids so the fields were conflicting.